axelrod/strategies/axelrod_second.py

```diff
@@ -98,6 +98,40 @@ def strategy(opponent: Player) -> Action:
         return random_choice(1 - defection_prop)
 
 
+class Grisell(Player):
+    """
+    Submitted to Axelrod's second tournament by Gail Grisell.  It came 23rd was written in 10 lines of BASIC.
```

Comment:
`and was written`

Also can you return at 80 characters please? (same for rest of docstring)